care/emr/tests/test_patient_api.py

```
@@ -206,6 +206,42 @@ def test_invalid_date_of_birth_and_death_date(self):
+        patient_id = response.data["id"]
+
+        # Update patient to set deceased_datetime to None
+        update_url = reverse("patient-detail", kwargs={"external_id": patient_id})
+        patient_data["deceased_datetime"] = None
+        response = self.client.put(update_url, patient_data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertIsNone(response.data["deceased_datetime"])
+
+    def test_non_superuser_cannot_unmark_deceased(self):
+        """Test that a non-superuser cannot set deceased_datetime back to None"""
+        geo_organization = self.create_organization(org_type="govt")
```

Ruff (0.15.2)

[error] Local variable `geo_organization` is assigned to but never used

Remove assignment to unused variable `geo_organization`

(F841)